### PR TITLE
Simple helper library to decode bytes to uint64 array

### LINKED ISSUE #401: Simple helper library to decode bytes to `uint64[]`
- [ ] Create library (`bandchain-helper-library`)
- [ ] Implement ```toUint64List(bytes memory) returns(uint64[])```
- [ ] Deploy library to npm (look example in `band-solidity` in band repo)

### PR DESCRIPTION
**Pull request by prin-r**
Fix #401

### CODE REVIEW

File: bridges/evm/contracts/BandChainLib.sol [hunk 1]
```diff
@@ -0,0 +1,18 @@
+pragma solidity 0.5.14;
+
+library BandChainLib {
+  function toUint64List(bytes memory _data) internal pure returns(uint64[] memory) {
+    uint64[] memory result = new uint64[](_data.length / 8);
+    require(result.length * 8 == _data.length, 'DATA_LENGTH_IS_INVALID');
+    
+    for (uint256 i = 0; i < result.length; i++) {
+      bytes8 tmp;
```
Comment (sorawit) [inline bridges/evm/contracts/BandChainLib.sol]: let's make this `uint64` and update line 11 appropriately.

File: bridges/evm/contracts/BandChainLib.sol [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      assembly {
+        tmp := mload(add(_data, add(0x20, mul(i,0x08))))
```
Comment (sorawit) [inline bridges/evm/contracts/BandChainLib.sol]: nit: space after comma

File: bridges/evm/package.json [hunk 3]
```diff
@@ -1,11 +1,25 @@
...
     "@openzeppelin/test-helpers": "^0.5.4",
+    "bandchain-helper-library": "^0.0.5",
     "chai": "^4.2.0",
     "openzeppelin-solidity": "^2.4.0",
     "openzeppelin-test-helpers": "^0.5.1",
     "truffle": "^5.1.3",
     "truffle-flattener": "^1.4.2"
   },
-  "version": "0.0.0"
+  "license": "MIT"
```
Comment (sorawit) [inline bridges/evm/package.json]: `Apache-2.0`

File: bridges/evm/yarn.lock [hunk 4]
```diff
@@ -256,6 +256,24 @@ balanced-match@^1.0.0:
   resolved "https://registry.yarnpkg.com/balanced-match/-/balanced-match-1.0.0.tgz#89b4d199ab2bee49de164ea02b89ce462d71b767"
   integrity sha1-ibTRmasr7kneFk6gK4nORi1xt2c=
 
+bandchain-helper-library@^0.0.4:
```
Comment (sorawit) [inline bridges/evm/yarn.lock]: Remove?

**Review by sorawit — CHANGES_REQUESTED** [review]

File: bridges/evm/contracts/BandChainLib.sol (hunk 1 above)
Comment (prin-r) [reply bridges/evm/contracts/BandChainLib.sol]: done

File: bridges/evm/package.json (hunk 3 above)
Comment (prin-r) [reply bridges/evm/package.json]: done

File: bridges/evm/yarn.lock (hunk 4 above)
Comment (prin-r) [reply bridges/evm/yarn.lock]: done

File: bridges/evm/contracts/BandChainLib.sol (hunk 2 above)
Comment (prin-r) [reply bridges/evm/contracts/BandChainLib.sol]: done

**Review by sorawit — APPROVED** [review]

**Review by taobun — APPROVED** [review]

**Pull request merged**